### PR TITLE
Added citation for Locations. 

### PR DESCRIPTION
**Pull request by eddiejibson**

### CODE REVIEW

File: Hosting/solutions/DigitalOcean.json [hunk 1]
```diff
@@ -33,7 +33,10 @@
 			"support": 1
 		},
 		"locations": {
-			"info": "8 Locations. Citation needed."
+			"info": "8 Locations. Citation needed.",
```
Comment (timmyRS) [inline Hosting/solutions/DigitalOcean.json]: Citation is no longer needed.

File: Hosting/solutions/Scaleway.json [hunk 2]
```diff
@@ -26,7 +26,10 @@
 			"support": 1
 		},
 		"locations": {
-			"info": "2 Locations. Citation needed."
+			"info": "2 Locations. Citation needed.",
```
Comment (timmyRS) [inline Hosting/solutions/Scaleway.json]: Citation is no longer needed.

File: Hosting/solutions/Linode.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
 		"locations": {
 			"info": "8 Locations",
 			"cites": [
+				"https://blog.linode.com/category/datacenters/",
```
Comment (timmyrs) [inline Hosting/solutions/Linode.json]: Twice?

**Comment by eddiejibson** [comment]
try now yo

**Review by timmyRS — CHANGES_REQUESTED** [review]

File: Hosting/solutions/Linode.json [hunk 4]
```diff
@@ -34,6 +34,7 @@
...
 				"https://blog.linode.com/category/datacenters/"
```
Comment (timmyRS) [inline Hosting/solutions/Linode.json]: Twice?

**Comment by eddiejibson** [comment]
all sorted now

**Review by timmyRS — APPROVED** [review]

**Pull request merged**